app/pb_kits/playbook/pb_typeahead/typeahead.rb

```diff
@@ -25,6 +32,22 @@ def data
           pb_typeahead_kit_search_debounce_timeout: search_debounce_timeout
         )
       end
+
+      def typeahead_with_pills_options
+        base_options = {
+          isMulti: true,
+          label: label,
+          options: options,
+          placeholder: placeholder
+        }
+
+        base_options.merge!({
+          async: true,
+          asyncUrl: async_url
+        }) if async
+
+        base_options
+      end
```

Comment:
This provides the `props` hash for passing object `props` to the React kit instances